### PR TITLE
FIX-TEMPLATE: 201-private-aks-cluster

### PR DESCRIPTION
**Pull request by olga-spektra**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: 201-private-aks-cluster/azuredeploy.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
       "defaultValue": "[concat('aks-', uniqueString(resourceGroup().id))]",
       "type": "string",
       "metadata": {
-        "description": "Specifies the name of the AKS cluster."
+        "description": "Specifies the name of the AKS cluste."
```
Comment (bmoore-msft) [inline 201-private-aks-cluster/azuredeploy.json:16]: revert this change

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 201-private-aks-cluster/azuredeploy.json (hunk 1 above)
Comment (olga-spektra) [reply 201-private-aks-cluster/azuredeploy.json]: done :)
